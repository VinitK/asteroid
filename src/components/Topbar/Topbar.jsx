import React from 'react';
import { Link } from 'react-router-dom';

import './Topbar.css';

const Topbar = () => {
    return (
        <div className="Topbar w100pc">
            <div className="frow fjcsb faic w80pc ma">
                <h2><Link to="/" className="link">Asteroids</Link></h2>
                <Link to="/" className="link">Search</Link>
            </div>
        </div>
    );
};

export default Topbar;