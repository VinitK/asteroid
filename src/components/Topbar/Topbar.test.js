import React from 'react';
import { render } from '@testing-library/react';
import { shallow } from 'enzyme';
import Topbar from './Topbar';

describe("Topbar Testing with Jest", () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallow(<Topbar />);

    });
    it('Topbar Component Snapshot', () => {
        const wrapper = shallow(<Topbar />);
        expect(wrapper).toMatchSnapshot();
    })
});
