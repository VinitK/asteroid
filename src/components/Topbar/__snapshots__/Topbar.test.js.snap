// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Topbar Testing with Jest Topbar Component Snapshot 1`] = `ShallowWrapper {}`;
